config.js

```diff
@@ -334,6 +313,33 @@ exports.updateFromArgv = function() {
       ["minperiodtimeunranked", "minperiodtimeblitzunranked, --minperiodtimeliveunranked and/or --minperiodtimecorrunranked"]
       ]
     deprecatedArgs.forEach(ar => testDeprecated(...ar))
+
+    let familyArray = []; 
+    function familyArrayFromGeneralArg(generalArg) {
```

Comment:
I wonder if the function should return an array?

Reply:
interesting idea, i didnt think of it enough

Reply:
done by dorus code :+1: